CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `DiscreteParameter.transform` 
 - `Surrogate` models now operate on dataframes in experimental representation instead of
   tensors in computational representation
+- `Surrogate.posterior` models now returns a `Posterior` object
```

Comment:
It seems like you still need to delete the custom surrogate examples since the "Build docs" pipeline still tries to build them.